lib/matplotlib/axes/_axes.py

```diff
@@ -1608,21 +1609,23 @@ def semilogx(self, *args, **kwargs):
     # @_preprocess_data() # let 'plot' do the unpacking..
     @docstring.dedent_interpd
     def semilogy(self, *args, **kwargs):
-        r"""Make a plot with log scaling on the `y` axis.
+        r"""
+        Make a plot with log scaling on the *y* axis.
```

Comment:
This is should be reverted it back to backticks (sorry :( we are trying to make things slightly more consistent across the library).